tailwind.config.js

```diff
@@ -1,6 +1,5 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  prefix: "tw-",
```

Comment:
Please do not remove it! other parts used this config option

Reply:
@anmode if I add this to the tailwind configuration file my style code will be deleted. ![Screenshot (425)](https://github.com/anmode/grabtern-frontend/assets/106610322/f86db148-62d7-433e-a29b-3d005cf584ec)

Reply:
@anmode ok so I have to add tw- in front of tailwind classes

Reply:
yes yeS!